apps/api-extractor-model/src/model/ApiTypeAlias.ts

```diff
@@ -42,13 +53,29 @@ export interface IApiTypeAliasOptions extends
  *
  * @public
  */
-export class ApiTypeAlias extends ApiNameMixin(ApiReleaseTagMixin(ApiDeclaredItem)) {
+export class ApiTypeAlias extends ApiTypeParameterListMixin(ApiNameMixin(ApiReleaseTagMixin(ApiDeclaredItem))) {
+  /**
+   * An {@link Excerpt} that describes the type of the alias.
+   */
+  public readonly aliasTypeExcerpt: Excerpt;
```

Comment:
I find the naming `...TypeAlias.aliasType...` to be somewhat uninformative.  It seems to muddy the waters of whether the "alias" is the thing to the left or the right of the `=`.

Could we rename this to `originalTypeExcerpt`?  Seems more straightforward to distinguish `type TheAlias = TheOriginalType`.

Reply:
That doesn't align with the fact that type aliases can have type arguments, since it's not an "original" type, per se. If anything I would just rename it to `typeExcerpt` (dropping the "alias" bit).

Reply:
It also mostly aligns with the fact that an `ApiVariable` as a `variableTypeExcerpt`. If I used that naming scheme it would have been `TypeAlias.typeAliasType`. Of the three (`aliasTypeExcerpt`, `typeExcerpt` or `typeAliasTypeExcerpt`), which would you prefer?

Reply:
But might we later introduce an excerpt corresponding to the `Boxed<T>` part (to the left of the `=`)?  That thing is also a "type", at least to a layperson.  It would be nice to have a more specific jargon.  I can't think of anything very good, though.  (alias "bodyExcerpt", alias "definitionExcerpt", ...)

If you can't think of anything better, `type` is better than `aliasType`.

Reply:
An ApiTypeAlias is also an ApiTypeParameterListMixin, so for `type Boxed<T> = { box: T; };`, the `<T>` will be found in `ApiTypeAlias#typeParameters` and the `{ box: T; }` will be found in `ApiTypeAlias#typeExcerpt` (or whatever its called). I think the whole of the declaration is covered.

Reply:
> Of the three (`aliasTypeExcerpt`, `typeExcerpt` or `typeAliasTypeExcerpt`), which would you prefer?

Definitely `typeExcerpt` of those three.  Throwing in more copies of the words "alias" and/or "type" are definitely not clarifying anything heheh.

For `type A = B;" I'm somewhat comfortable telling people that "`A` is the alias, and `B` is the type."